examples/java_postgresql_mixed_binding_type/README.md

```diff
@@ -0,0 +1,269 @@
+
+``` shell
+make install-service-binding-operator-master
+```
+
+#### Install the DB operator using an `OperatorSource`
+
+Apply the following `OperatorSource`:
+
+```shell
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: db-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: pmacik
+EOS
+```
+
+Alternatively, you can perform the same task with this make command:
+
+```shell
+make install-backing-db-operator-source
+```
+
+Then navigate to the `Operators`->`OperatorHub` in the OpenShift console and in the `Database` category select the `PostgreSQL Database` operator
+
+![PostgreSQL Database Operator as shown in OperatorHub](../../assets/operator-hub-pgo-screenshot.png)
+
+and install a `stable` version.
+
+This makes the `Database` custom resource available, that the application developer will use later.
+
+### Application Developer
+
+#### Create a namespace called `service-binding-demo`
+
+The application and the DB needs a namespace to live in so let's create one for them:
+
+```shell
+cat <<EOS |kubectl apply -f -
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: service-binding-demo
+EOS
+```
+
+Alternatively, you can perform the same task with this make command:
+
+```shell
+make create-project
+```
+
+#### Import an application
+
+In this example we will import an arbitrary [Java Spring Boot application](https://github.com/ldimaggi/java-rest-http-crud).
+
+In the OpenShift Console switch to the Developer perspective. (Make sure you have selected the `service-binding-demo` project). Navigate to the `+ADD` page from the menu and then click on the `[Import from Git]` button. Fill in the form with the following:
+
+* `Git Repo URL` = `https://github.com/ldimaggi/java-rest-http-crud`
+* `Project` = `service-binding-demo`
+* `Application`->`Create New Application` = `java-app`
+* `Name` = `java-app`
+* `Builder Image` = `Java`
+* `Create a route to the application` = checked
+* `Select the resource type to generate` = Deployment
+
+and click on the `[Create]` button.
+
+Notice, that during the import no DB config was mentioned or requestd.
+
+When the application is running navigate to its route to verify that it is up. Try the application's UI to add a fruit - it causes an error proving that the DB is not connected.
+
+#### Create a DB instance for the application
+
+Now we utilize the DB operator that the cluster admin has installed. To create a DB instance just create a `Database` custom resource in the `service-binding-demo` namespace called `db-demo`:
+
+```shell
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: postgresql.baiju.dev/v1alpha1
+kind: Database
+metadata:
+  name: db-demo
+  namespace: service-binding-demo
+spec:
+  image: docker.io/postgres
+  imageName: postgres
+  dbName: db-demo
+  tags:
+	- "centos7-12.3"
+        - "centos7-12.4"
+  storage:
+    - primaryStorage:
```

Comment:
Perhaps this can be changed from `[]map[string]interface{}` to `map[string]interface{}` only:

```yaml
storage:
  primaryStorage:
    - storageClass
    - 1000
```

Although I'm blind on this one since I didn't understand the data-is `storageClass` associated with the `1000` value or not?

Reply:
@isutton I just want to depict that we can fetch both string `storageClass` and int `1000` to demonstrate mixed binding type